<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixel art</string>
  </property>
  <widget class="QLabel" name="label_filename">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>45</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_contents">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>680</width>
     <height>280</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="Output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>680</width>
     <height>300</height>
    </rect>
   </property>
   <property name="title">
    <string>IMAGE</string>
   </property>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>660</width>
      <height>280</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>280</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>660</width>
     <height>35</height>
    </rect>
   </property>
   <property name="title">
    <string>Graphic mode</string>
   </property>
   <widget class="QRadioButton" name="btn_pixel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixel ART</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_xls">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>12</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>XLS      Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_size">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>12</y>
      <width>50</width>
      <height>18</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>680</width>
     <height>30</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>7</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>grey</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>7</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>0</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>0</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>0</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>Output</zorder>
  <zorder>label_filename</zorder>
  <zorder>pushButton_open</zorder>
  <zorder>textEdit_contents</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_start</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_stop</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_open</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>loadImageFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>rad_grey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>653</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>rad_color()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
  <slot>stop()</slot>
  <slot>slot_fileopen()</slot>
  <slot>loadImageFromFile()</slot>
  <slot>clear()</slot>
  <slot>rad_grey()</slot>
  <slot>rad_color()</slot>
  <slot>img2xls()</slot>
  <slot>img2ascii()</slot>
 </slots>
</ui>
